frontend/public/components/sidebars/resource-quota-sidebar.jsx

```diff
@@ -0,0 +1,36 @@
+import * as _ from 'lodash-es';
+import * as React from 'react';
+
+import { ResourceQuotaModel } from '../../models';
+import { referenceForModel } from '../../module/k8s';
+import { SampleYaml } from './resource-sidebar';
+
+const samples = [
+  {
+    highlightText: 'Specify',
```

Comment:
If we use highlightText, I'd think we'd want to consistently use that for each example. I'm not sure it makes sense in this case, though.

Reply:
I was playing with the different tags but I should make these consistent.